### ISSUE TITLE
Some difference

### ISSUE THREAD

**Issue by shentaowang** [truskovskiyk/nima.pytorch#5]
Thank you very much for sharing. First I used your mobilenet and weights, then I used the mobilenet and weights https://github.com/titu1994/neural-image-assessment share.I found there is a big difference between the two on the same data set. Can you compare your work to the results given in the paper?

**Comment by truskovskiyk** [comment]
Hi @GeniusLight
Thanks for the question! 
Yes, there might be a huge difference, it depends on  

- model
I use mobile-net-v2 (https://arxiv.org/abs/1801.04381) in https://github.com/titu1994/neural-image-assessment implementation author use mobile-net-v1 (https://arxiv.org/abs/1704.04861) and some other models
- optimization and hyperparameters tyning (learning rate, num epoch, augmentation, etc)
- the randomness of the training algorithm
- different train/validation dataset split

please, could you provide more information about differences?

**Comment by truskovskiyk** [comment]
@GeniusLight about comparing results with the paper (https://arxiv.org/pdf/1709.05424.pdf)
Here the table with author results 
<img width="433" alt="screen shot 2018-04-09 at 4 20 16 pm" src="https://user-images.githubusercontent.com/7893705/38500044-fa165814-3c11-11e8-8310-de4f44ee032d.png">

I got on test dataset EMD  =  0.080
But we have different train/validation/test split

**Comment by shentaowang** [comment]
I want to use this method to evaluate the effect of converting a grayscale image into a color image.I evaluate 120 images.
When use you model and weight, the meas score for grayscale(copy two channels) is 4.18831566, the means score for pesudo rgb color is 3.84607850. When use the mobile-net and weights https://github.com/titu1994/neural-image-assessment gave, the score is 4.18837916 and 4.76663283. The result is contradictory. I'm not quite sure if the degree of differentiation of this score is significant
On 4/9/2018 21:18，Kyryl Truskovskyi<notifications@github.com> wrote：

Hi @GeniusLight
Thanks for the question!
Yes, there might be a huge difference, it depends on

model
I use mobile-net-v2 (https://arxiv.org/abs/1801.04381) in https://github.com/titu1994/neural-image-assessment implementation author use mobile-net-v1 (https://arxiv.org/abs/1704.04861) and some other models
optimization and hyperparameters tyning (learning rate, num epoch, augmentation, etc)
the randomness of the training algorithm
different train/validation dataset split

please, could you provide more information about differences?

**Comment by truskovskiyk** [comment]
@GeniusLight Now I understand what you mean. So I need some time to figure out and fix this problem. As soon I will find the problem I will posts results
